common-api/src/main/scala/pl/touk/nussknacker/engine/graph/expression/Expression.scala

```diff
@@ -1,17 +1,32 @@
 package pl.touk.nussknacker.engine.graph.expression
 
 import io.circe.generic.JsonCodec
+import io.circe.syntax.EncoderOps
 
+// TODO in the future 'expression' should be a dedicated type rather than raw string, it would for example make DictLabelWithKeyExpression handling prettier
 @JsonCodec case class Expression(language: String, expression: String)
 
 object Expression {
 
   object Language {
-    val Spel         = "spel"
-    val SpelTemplate = "spelTemplate"
+    val Spel             = "spel"
+    val SpelTemplate     = "spelTemplate"
+    val Literal          = "literal" // TODO remove if not needed?
+    val DictLabelWithKey = "dictLabelWithKey"
```

Comment:
Can we stick with key -> label order. Do you feal that the reversed order in this names is somehow very important? key -> label seems more natural